.github/file-filters.yml

```diff
@@ -12,6 +12,7 @@ stack:
   - "**.mako"
   - "**.lock"
   - "**.json"
+  - notebooks/api/**.ipynb
```

Comment:
@shubham3121 , made a small change here from `notebooks/api/**/*.ipynb` to `notebooks/api/**.ipynb` as in the 0.9 notebooks, 

we have projects as a separate folder

notebooks
   - api
       - 0.9
             -projects.ipynb
to capture all the notebooks